src/TelnetClient.php

```diff
@@ -52,6 +52,11 @@ class TelnetClient implements TelnetClientInterface
      */
     protected $lineEnding = "\n";
 
+    /**
+     * @var int
+     */
+    protected $maxReadCharacters = 0;
```

Comment:
```suggestion
    protected $maxReadBytes = 0;
```

Socket reading reads bytes and not characters.

Reply:
I went with character because that was already used as what was being read.
I've renamed all instances: 3dd864d